test/opsem/muchang.gstr.ll

```diff
@@ -1,6 +1,5 @@
-; RUN: %seabmc "%s" 2>&1 | %oc %s
+; RUN: %seabmc --horn-bv2-word-size=1 "%s" 2>&1 | %oc %s
```

Comment:
TODO: load/store instructions are not word-aligned, this arg will be removed once that is implemented